scripts/album.js

```diff
@@ -51,7 +51,58 @@ var createSongRow = function(songNumber, songName, songLength) {
   + '</tr>'
   ;
 
-  return $(template);
+  var $row = $(template);
+
+  var clickHandler = function() {
+    var $songItem = $(this).find('.song-item-number');
+    var $songItemNumber = $songItem.attr('data-song-number');
+
+    // if there isn't a song playing, set the clicked song item number to pause
+    if (currentlyPlayingSong === null) {
```

Comment:
So this right here I would expect to handle it the transition. Problem is, we're not even getting this far. I think the problem is upstream?